### PR TITLE
STYLING-DONE

### PR DESCRIPTION
**Pull request by pratap-panabaka**
This is the last project for the Bookstore website. Now that all the functionality is done, Have to add styling. Use Zeplin as a collaboration tool.

### CODE REVIEW

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @PRATAP-KUMAR , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
- [ ] The right side user_icon is missing, Kindly add it.
- [ ] The line & content of percentage is missing, It should be same as in requirement.
- [ ] there is one line seperater above the add book form.
- [ ] the add book form should be float to the screen width kindly fix it.

- Your website
Screenshot from 2022-09-23 19-10-55

- Originnal website
Screenshot from 2022-09-23 19-15-55

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/css/Book.css [hunk 1]
```diff
@@ -1,46 +1,98 @@
 .BookCard {
-  width: 75rem;
-  height: 2.625rem;
-  margin: 2.313rem 6.25rem 0;
```
Comment (BenMukebo) [inline src/css/Book.css:4]: - [OPTIONAL] Make sure all line bars are aligned in the same line as shown on the image below 👇🏻

File: src/css/Book.css [hunk 2]
```diff
@@ -1,46 +1,98 @@
...
+  border: 1px solid #0290ff;
+}
+
+.one {
+  margin-bottom: 0.75rem;
 }
 
-.BookCardContainer {
+.two {
   display: flex;
```
Comment (BenMukebo) [inline src/css/Book.css:20]: - [OPTIONAL] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code to No repeating of the same rule by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```css
.d-flex {
  display: flex;
}

.flex-center {
  align-items: center;
  justify-content: center;
}
```
- JS file
```js
<div className="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉

**Review by BenMukebo — APPROVED** [review]
### Hi @PRATAP-KUMAR, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



### To Highlight :1st_place_medal: 
- Your README is description. ✔️  
- No Linter errors :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/css/Book.css (hunk 2 above)
Comment (pratap-panabaka) [reply src/css/Book.css:20]: @BenMukebo Thank you very much for giving me this knowledge, I will try to understand this code and try to implement it.
Thank you very much once again for the suggesions.

**Pull request merged**